app/assets/javascripts/media/show.js

```diff
@@ -1,13 +1,19 @@
 $( function ( ) {
-  $( "a.hidden-image" ).on( "click", function ( e ) {
+  $( "a.hidden-media-link" ).on( "click", function ( e ) {
     e.preventDefault( );
     // open a new window now in the context of the user action so it doesn't get popup-blocked
     var resourceWindow = window.open( );
+    var resource_data;
```

Comment:
I would recommend enabling automated linting in the whatever editor you are using. We have an [.eslintrc](https://github.com/inaturalist/inaturalist/blob/a6ed12941087f39d446bb6b585912944d440d9ee/app/assets/javascripts/.eslintrc) config for ESLint specifying some preferred ways to format JS. There are a few ways the changes here conflict with those rules. If your editor cannot implement in-line linting (this is a very useful feature that I'd highly recommend), you can manually run eslint with e.g. `node_modules/.bin/eslint app/assets/javascripts/media/show.js`. Could you please address the errors ESLint is reporting?